plugin-api/src/main/java/org/hyperledger/besu/plugin/services/metrics/IbftQueries.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.hyperledger.besu.plugin.services.metrics;
+
+import org.hyperledger.besu.plugin.data.Address;
+
+import java.util.Collection;
+
+public interface IbftQueries extends PoAMetricsService {
```

Comment:
Given this is part of the plugin apis it will need comprehensive javadoc.

Reply:
Agreed - for now just needed to get the code in front of Uzi as he's working on something related.